File: crowdRiseWeb-master/crowdRiseWeb-master/src/CrowdRiseBundle/Entity/Signalisation.php
<?php

namespace CrowdRiseBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Signalisation
 *
 * @ORM\Table(name="signalisation", indexes={@ORM\Index(name="id_commentaire", columns={"id_commentaire"}), @ORM\Index(name="id_membre", columns={"id_membre"})})
 * @ORM\Entity
 */
class Signalisation
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="raison", type="string", length=100, nullable=false)
     */
    private $raison;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var \Commentaire
     *
     * @ORM\ManyToOne(targetEntity="Commentaire")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_commentaire", referencedColumnName="id")
     * })
     */
    private $idCommentaire;

    /**
     * @var \Membre
     *
     * @ORM\ManyToOne(targetEntity="Membre")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_membre", referencedColumnName="id")
     * })
     */
    private $idMembre;

    function getId() {
        return $this->id;
    }

    function getRaison() {
        return $this->raison;
    }

    function getDate() {
        return $this->date;
    }

    function getIdCommentaire() {
        return $this->idCommentaire;
    }

    function getIdMembre() {
        return $this->idMembre;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setRaison($raison) {
        $this->raison = $raison;
    }

    function setDate(\DateTime $date) {
        $this->date = $date;
    }

    function setIdCommentaire(\Commentaire $idCommentaire) {
        $this->idCommentaire = $idCommentaire;
    }

    function setIdMembre(\Membre $idMembre) {
        $this->idMembre = $idMembre;
    }



}

File: CrowdRiseMobile-master/CrowdRiseMobile-master/src/Entity/Signalisation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entity;



/**
 *
 * @author Aymen
 */


public class Signalisation {
    private int id;
    private Membre membre;
    private Commentaire commentaire;
    private String raison;
    private String date;

    public Signalisation() {
    }

    public Signalisation(int id,Membre membre, Commentaire commentaire, String raison, String date) {
        this.id=id;
        this.membre = membre;
        this.commentaire = commentaire;
        this.raison = raison;
        this.date = date;
    }

    public Signalisation(Membre membre, Commentaire commentaire, String raison, String date) {
        this.membre = membre;
        this.commentaire = commentaire;
        this.raison = raison;
        this.date = date;
    }

    public Signalisation(int id) {
        this.id = id;
    }

    public Membre getMembre() {
        return membre;
    }

    public void setMembre(Membre membre) {
        this.membre = membre;
    }

    public Commentaire getCommentaire() {
        return commentaire;
    }

    public void setCommentaire(Commentaire commentaire) {
        this.commentaire = commentaire;
    }

    public String getRaison() {
        return raison;
    }

    public void setRaison(String raison) {
        this.raison = raison;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    
    public String toString() {
        return "Signalisation{" + "id=" + id + ", membre=" + membre + ", commentaire=" + commentaire + ", raison=" + raison + ", date=" + date + '}';
    }

    
    
    
    
}
